packages/vite-plugin-cloudflare/playground/prisma/prisma/schema.prisma

```diff
@@ -2,10 +2,15 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider        = "prisma-client-js"
-	previewFeatures = ["driverAdapters"]
-	// Default output directory does not work with Vite. See https://github.com/vitejs/vite/issues/19036#issuecomment-2558791944
-	output          = "../node_modules/@prisma/client-generated"
+  provider = "prisma-client"
+
+  runtime                = "workerd"
+  moduleFormat           = "esm"
+  generatedFileExtension = "ts"
+  importFileExtension    = "ts"
```

Comment:
@jamesopstad a small fyi, no action needed - I tried out the new `prisma-client` generator to see if this hanging issue would still happen now that we'd get an `import "<path>.wasm"` in the generated prisma code rather than an `import("<path>.wasm")`. Turns out it is indeed still an issue (it still hangs).

Reply:
Yea, I tried it today too, but no luck 🙈